editor/store/effects.js

```diff
@@ -288,36 +284,30 @@ export default {
 	},
 	SETUP_EDITOR( action ) {
 		const { post, settings } = action;
-		const effects = [];
 
 		// Parse content as blocks
+		let blocks = [];
```

Comment:
Incredibly minor: We could avoid wastefully allocating an array by moving this assignment into the final `else` and leaving as undefined:

```
let blocks;
if ( post.content.raw ) {
	// ...
} else if ( settings.template) {
	// ...
} else {
	blocks = [];
}